code/game/gamemodes/cult/cult_items.dm

```diff
@@ -367,11 +363,225 @@
+	return FALSE
+
+/obj/item/shield/mirror/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	var/turf/T = get_turf(hit_atom)
+	var/datum/thrownthing/D = throwingdatum
+	if(isliving(hit_atom))
+		var/mob/living/L = hit_atom
+		if(iscultist(L))
+			playsound(src, 'sound/weapons/throwtap.ogg', 50)
+			if(!L.restrained() && L.put_in_active_hand(src))
+				L.visible_message("<span class='warning'>[L] catches [src] out of the air!</span>")
+			else
+				L.visible_message("<span class='warning'>[src] bounces off of [L], as if repelled by an unseen force!</span>")
+		else if(!..())
+			if(!L.null_rod_check())
+				L.Weaken(1)
+				if(D?.thrower)
+					for(var/mob/living/Next in orange(2, T))
+						if(!Next.density || iscultist(Next))
+							continue
+						throw_at(Next, 3, 1, D.thrower)
+						return
+					throw_at(D.thrower, 7, 1, null)
+	else
+		..()
+
+
+/obj/item/twohanded/cult_spear
+	name = "blood halberd"
+	desc = "A sickening spear composed entirely of crystallized blood."
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "bloodspear0"
+	slot_flags = 0
+	force = 17
+	force_unwielded = 17
+	force_wielded = 24
+	throwforce = 40
+	throw_speed = 2
+	armour_penetration = 30
+	block_chance = 30
+	attack_verb = list("attacked", "impaled", "stabbed", "torn", "gored")
+	sharp = TRUE
+	no_spin_thrown = TRUE
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	var/datum/action/innate/cult/spear/spear_act
+
+/obj/item/twohanded/cult_spear/Destroy()
+	if(spear_act)
+		qdel(spear_act)
+	..()
+
+/obj/item/twohanded/cult_spear/update_icon()
+	icon_state = "bloodspear[wielded]"
+
+/obj/item/twohanded/cult_spear/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	var/turf/T = get_turf(hit_atom)
+	if(isliving(hit_atom))
+		var/mob/living/L = hit_atom
+		if(iscultist(L))
+			playsound(src, 'sound/weapons/throwtap.ogg', 50)
+			if(!L.restrained() && L.put_in_active_hand(src))
+				L.visible_message("<span class='warning'>[L] catches [src] out of the air!</span>")
+			else
+				L.visible_message("<span class='warning'>[src] bounces off of [L], as if repelled by an unseen force!</span>")
+		else if(!..())
+			if(!L.null_rod_check())
+				L.Weaken(3)
+			break_spear(T)
+	else
+		..()
+
+/obj/item/twohanded/cult_spear/proc/break_spear(turf/T)
+	if(src)
+		if(!T)
+			T = get_turf(src)
+		if(T)
+			T.visible_message("<span class='warning'>[src] shatters and melts back into blood!</span>")
+			new /obj/effect/temp_visual/cult/sparks(T)
+			new /obj/effect/decal/cleanable/blood/splatter(T)
+			playsound(T, 'sound/effects/glassbr3.ogg', 100)
+	qdel(src)
+
+/obj/item/twohanded/cult_spear/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(wielded)
+		final_block_chance *= 2
+	if(prob(final_block_chance))
+		if(attack_type == PROJECTILE_ATTACK)
+			owner.visible_message("<span class='danger'>[owner] deflects [attack_text] with [src]!</span>")

```

Comment:
Maybe add a message for the user as well

Reply:
This is consistent with how blocking show in chat for other items